Source/Menu/Options.Designer.cs

```diff
@@ -1194,11 +1194,6 @@ private void InitializeComponent()
             // numericWorldObjectDensity
             // 
             this.numericWorldObjectDensity.Location = new System.Drawing.Point(6, 299);
-            this.numericWorldObjectDensity.Maximum = new decimal(new int[] {
-            10,
-            0,
-            0,
-            0});
```

Comment:
Shouldn't there be a maximum set to 100?

Reply:
Good catch, Cedric.

Not sure what went wrong there, but I've used the time to re-consider.
I've now changed the default, the maximum and the Manual text.